### DISCUSSION TITLE
Brazilian Portuguese translation

### DISCUSSION THREAD

**Discussion by GuzMan2019** [jwallet/spy-spotify#469] [Ideas]
I suggest implementing a feature that makes it possible to translate the application into multiple languages. The file containing the translation can be a .ts or .ini as used in NirSoft tools (https://www.nirsoft.net/).

**Comment by jwallet** [comment]
I once added that feature but I removed it, it was too difficult to maintain it. A feature came out and I had to wait for translations that never came. So I can only translate it for FR and EN, I even think about removing FR. Translating takes too much time, time that can be used to fix bugs.

**Reply by maboroshin** [reply]
Translations was added in January 2021. https://github.com/jwallet/spy-spotify/pull/313/commits
It was removed in August 2021. https://github.com/jwallet/spy-spotify/commit/3c1764771f21bba73b0f5f378bd22a9072e845b6

I am a translation contributor, not a programmer.

I have seen projects that only update ```en.resx```. The translator only needs to translate ```en.resx``` into own language, e.g. ```fr.resx```. The developer updates only ```en.resx```. The translator will reflect the differences in ```fr.resx```.

The differences in Designer.cs are mostly comments. It seems to be difficult to maintain the ```summary``` tag that is a comment. Maintaining the ```summary``` tag appears unnecessary. Only the following lines actually differ. Would you be able to put them in one file?
```
internal class en
internal en()
EspionSpotify.Translations.es", typeof(en).Assembly
```
